src/main/java/step2/model/Point.java

```diff
@@ -0,0 +1,34 @@
+package step2.model;
+
+public class Point {
+
+    private boolean status;
+
+    public Point(boolean status) {
+        this.status = status;
+    }
+
+    public Point() {
```

Comment:
```suggestion
    public Point() {
        this(false);
    }
```

주생성자를 호출하게 한다면 필드에 대해 final 키워드 까지 추가 가능할것 같습니다.